setup.cfg

```diff
@@ -35,5 +35,6 @@ per-file-ignores =
     scripts/*.py:D100,D101,D102,D103,D104,D107,D205,D400,D401,RST201,RST205,RST301
     test/*.py:D104
     test/*/*.py:D100,D101,D102,D103,D104,D105,D107
+    test/unit/adversarial/*.py:A003,D100,D101,D102,D103,D104,D105,D106,D107
```

Comment:
why? :D

Reply:
Hihi yeah.... Lazy? In my defense most of these also appear in the other modules haha :)

Reply:
Just run `black` on your code.